fcd_weight_scale_mrp/models/res_conf_settings.py

```diff
@@ -13,3 +13,4 @@ class ResConfigSettings(models.TransientModel):
         readonly=False,
     )
     weight_scale_password = fields.Char(related='company_id.weight_scale_password', string="Password", readonly=False)
+    weight_scale_limit = fields.Float(related='company_id.weight_scale_limit', string="Limit", readonly=False)
```

Comment:
If you remove sting attribute, it will be obtained from res.company field, not tested